src/entities/ProductQuantityEntity.ts

```diff
@@ -0,0 +1,9 @@
+export class ProductQuantityEntity {
+    public id?: string;
+    public list_id?: string;
+    public product_id?: string;
+    public initial_quantity?: number;
+    public final_quantity: number;
+    public suggestion_quantity: number;
```

Comment:
**alteração**: aqui nesse caso acho que só `initial_quantity` deveria ser obrigatório.. porque tipo, quando a lista inicial for criada, a gente não vai ter uma sugestão ainda.